src/herder/Herder.h

```diff
@@ -55,6 +55,9 @@ class Herder
     // How many ledgers in the past we keep track of
     static uint32 const MAX_SLOTS_TO_REMEMBER;
 
+    // Min time between before first prepare (to avoid metric skewing)
+    static std::chrono::nanoseconds const NOMINATION_THRESHOLD_NANOSECS;
```

Comment:
comment doesn't really correspond to reality (it's not just about nomination):
this threshold is used to filter outliers when timing events.
I would just call it `TIMERS_THRESHOLD_NANOSEC`